.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 yarn-error.log
+*.DS_Store
```

Comment:
Not a problem to have this in this file (except it could be ordered alphabetically) but this should be in your `~/.gitignore`

Reply:
thx, i added it

Reply:
And i ordered it.